packages/screens/Multisig/MultisigCreateScreen.tsx

```diff
@@ -118,39 +120,73 @@ export const MultisigCreateScreen = () => {
       parseInt(signatureRequired, 10),
     );
 
-    const res = await multisigClient.CreateOrJoinMultisig({
-      authToken,
-      chainId: selectedNetwork.chainId,
-      bech32Prefix: selectedNetwork.addressPrefix,
-      multisigPubkeyJson: JSON.stringify(multisigPubkey),
-      name,
-    });
+    try {
+      const res = await multisigClient.CreateOrJoinMultisig({
+        authToken,
+        chainId: selectedNetwork.chainId,
+        bech32Prefix: selectedNetwork.addressPrefix,
+        multisigPubkeyJson: JSON.stringify(multisigPubkey),
+        name,
+      });
 
-    navigation.navigate("MultisigWalletDashboard", {
-      id: getUserId(selectedNetwork?.id, res.multisigAddress),
-    });
+      navigation.navigate("MultisigWalletDashboard", {
+        id: getUserId(selectedNetwork?.id, res.multisigAddress),
+      });
+    } catch (err) {
+      console.error("Failed to create multisig", err);
+      const message = err instanceof Error ? err.message : "";
```

Comment:
in case where err is not Error we can try to stringify the err (e.g: `${err}`) instead of sending ""

Reply:
https://github.com/TERITORI/teritori-dapp/pull/1354/commits/7666d919d204c3e6e70948b541ab6f42a76f0303